sdks/python/apache_beam/examples/cookbook/bigquery_tornadoes.py

```diff
@@ -87,14 +87,19 @@ def run(argv=None):
     rows = p | 'read' >> beam.io.Read(beam.io.BigQuerySource(known_args.input))
     counts = count_tornadoes(rows)
 
+    if p.options.get_all_options()['temp_location'].startswith('gs://'):
```

Comment:
Seems temp_location has to be a GCS path (https://github.com/apache/beam/blob/master/sdks/python/apache_beam/options/pipeline_options.py#L426). May not need `gs://` check.